STAR/2.4.2a/AlignReads.nf

```diff
@@ -0,0 +1,29 @@
+process alignReads {
+    tag {"STAR alignReads ${sample_id} - ${rg_id}"}
+    label 'STAR_2_4_2a'
+    label 'STAR_2_4_2a_alignReads'
+    container = '/hpc/cog_bioinf/ubec/tools/rnaseq_containers/star_2_4_2a-squashfs-pack.gz.squashfs'
+    clusterOptions = workflow.profile == "sge" ? "-l h_vmem=${params.star_mem}" : ""
+    shell = ['/bin/bash', '-euo', 'pipefail']
+
+    input:
+    tuple sample_id, rg_id, file(fastqs: "*") 
+    file(index)
+
+    output:
+    tuple sample_id, rg_id, file("${rg_id}.Aligned.sortedByCoord.out.bam" ),file("${rg_id}.Log.final.out"), file("${rg_id}.Log.out"), file("${rg_id}.SJ.out.tab")
```

Comment:
Are the log files required as an output and/or needed as an input for other tools? Maybe separate from bam output?

And also no index file (bai) created?